src/Sdk/WebApi/WebApi/Results/Contracts.cs

```diff
@@ -0,0 +1,36 @@
+namespace GitHub.Services.Results.Contracts
```

Comment:
move this file up in the folder structure.
`src/Sdk/Results/WebApi/Contract.cs`.
same for the `ResultsHttpClient.cs`